lib/ecto/adapters/mysql/connection.ex

```diff
@@ -14,6 +14,17 @@ if Code.ensure_loaded?(Mariaex.Connection) do
       Mariaex.Connection.start_link(opts)
     end
 
+    def query(conn, sqls, params, opts) when is_list(sqls) do
+      Enum.reduce sqls, {}, fn(sql, response) ->
```

Comment:
I would prefer not to but I can take a stab at it before merging the PR, so no worries. Just let me know when this is ready for review.